website/src/index.js

```diff
@@ -0,0 +1,17 @@
+// import React from "react";
```

Comment:
this file also looks like some js file that survived the ts transition ;-) .. is it necessary?